easybuild/easyconfigs/m/Molden/Molden-7.3_fix_arg_handling.patch

```diff
@@ -0,0 +1,22 @@
+diff -ruN molden7.3.orig/src/molden.f molden7.3/src/molden.f
```

Comment:
Could you please add a patch description here (maybe just copy the one from the previous version: https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/m/Molden/Molden-7.1_fix_arg_handling.patch)? Note that this will change the checksum of the file, so the easyconfig will have to be adjusted.